opennav_docking/src/navigator.cpp

```diff
@@ -52,12 +52,16 @@ void Navigator::deactivate()
 void Navigator::goToPose(
   const geometry_msgs::msg::PoseStamped & pose,
   const rclcpp::Duration & max_staging_duration,
+  std::function<bool()> isPreempted,
   bool recursed)
 {
+  auto node = node_.lock();
+
   Nav2Pose::Goal goal;
   goal.pose = pose;
   goal.behavior_tree = navigator_bt_xml_;
   const auto timeout = max_staging_duration.to_chrono<std::chrono::milliseconds>();
```

Comment:
This is now unused

Reply:
It's not, we use the clock to check for timeout